posthog/api/test/__snapshots__/test_dashboard.ambr

```diff
@@ -724,7 +1087,177 @@
+  FROM "posthog_dashboardtile"
+  WHERE ("posthog_dashboardtile"."dashboard_id" = 2
+         AND "posthog_dashboardtile"."insight_id" = 2)
+  LIMIT 21
+  '
+---
+# name: TestDashboard.test_adding_insights_is_not_nplus1_for_gets.42
+  '
+  SELECT "posthog_dashboardtile"."id",
+         "posthog_dashboardtile"."dashboard_id",
+         "posthog_dashboardtile"."insight_id",
+         "posthog_dashboardtile"."layouts",
+         "posthog_dashboardtile"."color",
+         "posthog_dashboardtile"."filters_hash"
+  FROM "posthog_dashboardtile"
+  WHERE ("posthog_dashboardtile"."dashboard_id" = 2
+         AND "posthog_dashboardtile"."insight_id" = 2)
+  ORDER BY "posthog_dashboardtile"."id" ASC
+  LIMIT 1
+  '
+---
+# name: TestDashboard.test_adding_insights_is_not_nplus1_for_gets.43
+  '
+  SELECT "posthog_dashboardtile"."id",
+         "posthog_dashboardtile"."dashboard_id",
+         "posthog_dashboardtile"."insight_id",
+         "posthog_dashboardtile"."layouts",
+         "posthog_dashboardtile"."color",
+         "posthog_dashboardtile"."filters_hash"
+  FROM "posthog_dashboardtile"
+  WHERE ("posthog_dashboardtile"."dashboard_id" = 2
+         AND "posthog_dashboardtile"."insight_id" = 2)
+  ORDER BY "posthog_dashboardtile"."id" ASC
+  LIMIT 1
+  '
+---
+# name: TestDashboard.test_adding_insights_is_not_nplus1_for_gets.44
+  '
+  SELECT "posthog_dashboardtile"."id",
+         "posthog_dashboardtile"."dashboard_id",
+         "posthog_dashboardtile"."insight_id",
+         "posthog_dashboardtile"."layouts",
+         "posthog_dashboardtile"."color",
+         "posthog_dashboardtile"."filters_hash"
+  FROM "posthog_dashboardtile"
+  WHERE ("posthog_dashboardtile"."dashboard_id" = 2
+         AND "posthog_dashboardtile"."insight_id" = 2)
+  LIMIT 21
+  '
+---
+# name: TestDashboard.test_adding_insights_is_not_nplus1_for_gets.45
+  '
+  SELECT "posthog_dashboardtile"."id",
+         "posthog_dashboardtile"."dashboard_id",
+         "posthog_dashboardtile"."insight_id",
+         "posthog_dashboardtile"."layouts",
+         "posthog_dashboardtile"."color",
+         "posthog_dashboardtile"."filters_hash"
+  FROM "posthog_dashboardtile"
+  WHERE ("posthog_dashboardtile"."dashboard_id" = 2
+         AND "posthog_dashboardtile"."insight_id" = 2)
+  ORDER BY "posthog_dashboardtile"."id" ASC
+  LIMIT 1
+  '
+---
+# name: TestDashboard.test_adding_insights_is_not_nplus1_for_gets.46
+  '
+  SELECT "posthog_dashboardtile"."id",
+         "posthog_dashboardtile"."dashboard_id",
+         "posthog_dashboardtile"."insight_id",
+         "posthog_dashboardtile"."layouts",
+         "posthog_dashboardtile"."color",
+         "posthog_dashboardtile"."filters_hash"
+  FROM "posthog_dashboardtile"
+  WHERE ("posthog_dashboardtile"."dashboard_id" = 2
+         AND "posthog_dashboardtile"."insight_id" = 2)
+  ORDER BY "posthog_dashboardtile"."id" ASC
+  LIMIT 1
+  '
+---
+# name: TestDashboard.test_adding_insights_is_not_nplus1_for_gets.47
```

Comment:
Loads more queries in the `test_adding_insights_is_not_nplus1_for_gets` test. Is that intentional?